cohort-evaluator-spark/src/main/java/com/ibm/cohort/cql/spark/data/SparkSchemaCreator.java

```diff
@@ -234,7 +255,14 @@ else if (elements.size() > 1) {
 		}
 		return new Tuple2<>(contextDefinition.getPrimaryKeyColumn(), keyType);
 	}
-	
+
+	private List<ClassInfo> getClassInfos(String primaryDataType, ModelInfo modelInfo) {
+		return modelInfo.getTypeInfo().stream()
+				.map(x -> (ClassInfo) x)
```

Comment:
Similar casting concern from `SparkCqlEvaluator`.

Pedantic: You also used a different way to cast between here and `SparkCqlEvaluator`.

Reply:
addressed in dee4b3f